docs/platforms/javascript/common/performance/instrumentation/performance-metrics.mdx

```diff
@@ -34,6 +34,12 @@ Currently, unit conversion is only supported once the data has already been stor
 
 </Note>
 
+<Note>
+
+You should not rely on the transaction (`getTransaction` is marked as deprecated), but just use `startSpan()` [APIs](https://docs.sentry.io/platforms/javascript/guides/solid/performance/instrumentation/custom-instrumentation/#start-span) instead.
```

Comment:
Can you update the example used in line 29 to remove `getTransaction`? It'd be annoying for me if I used that example and then saw this...

Reply:
yes sure... but I am not confident how to properly use the `startSpan` in those particular examples...